catalog.html

```diff
@@ -0,0 +1,301 @@
+                                    Мотель
+                                </label>
+                            </li>
+                            <li class="catalog-filter-item">
+                                <label class="control">
+                                    <input class="control-item" type="radio" name="housing-group" value="apartments">
+                                    Апартаменты
+                                </label>
+                            </li>
+                        </ul>
+                    </fieldset>
+                    <fieldset class="catalog-filter-group">
+                        <legend class="catalog-filter-title">Стоимость, ₽:</legend>
+                        <label class="price">
+                        <input type="range" id="price" name="price" min="0" max="12000" value="0">от
+                        </label>
+                        <label class="price">
+                        <input type="range" id="max-price" name="price" min="0" max="12000" value="9000">до
+                        </label>
+                    </fieldset>
+                    <button class="button" type="submit">Применить</button>
+                    <button class="reset" type="reset">Сбросить</button> 
+                </form>
+                <h2>Найдено гостиниц: 38</h2>
+                <select class="select-control">
+                    <option value="poular">Сначала поуплярные</option>
+                    <option value="cheap">Сначала дешёвые</option>
+                    <option value="expensive">Сначала дорогие</option>                    
+                </select>
+                <a class="button button-checked" href="#">
+                    <span class="visually-hidden">Показать карточки</span>
+                </a>
+                <a class="button" href="#">
+                    <span class="visually-hidden">Показать слайды</span>
+                </a>
+                <a class="button" href="#">
+                    <span class="visually-hidden">Показать список</span>
+                </a>
+                <ul class="hotels-list">
+                    <li class="hotel-card">
+                        <a class="hotel-card-link" href="#">
+                            <img class="hotel-card-image" src="" alt="Описание картинки">
+                            <h3 class="hotel-card-title">Amara Resort &amp; Spa</h3>
+                        </a>
+                        <p class="housing-group">Гостиница</p>
+                        <p class="hotel-card-price">От 4000 ₽</p>
+                        <a class="detail" href="#">Подробнее</a>
+                        <a class="favorites" href="#">В избранное</a>
+                        <img class="star" src="" alt="Описание картинки">
+                        <img class="star" src="" alt="Описание картинки">
+                        <img class="star" src="" alt="Описание картинки">
+                        <img class="star" src="" alt="Описание картинки">
```

Comment:
Пока пиши просто
```suggestion
                        <span>4 звезды</span>
```

И прочее. Я потом сделаю демку с разными вариантами

Reply:
спану класс задавать?

Reply:
@Alekzander87 да, можно. Сейчас все равно стилизации нет, поэтому непринципиально